static/js/stream_data.js

```diff
@@ -392,6 +393,14 @@ exports.receives_audible_notifications = function (stream_name) {
     return sub.audible_notifications;
 };
 
+exports.receives_push_notifications = function (stream_name) {
+    var sub = exports.get_sub(stream_name);
+    if (sub === undefined) {
+        return false;
+    }
+    return sub.push_notifications;
+};
```

Comment:
This doesn't seem to be called?  But just as a note, we're trying to migrate away from accessing objects by `stream_name` wherever we can use `stream_id` instead.